src/DynamoCoreWpf/Views/CodeCompletion/CodeCompletionEditor.xaml.cs

```diff
@@ -418,4 +419,4 @@ internal void Dispose()
             this.dynamoViewModel.PreferencesViewModel.PropertyChanged -= OnPreferencesViewModelPropertyChanged;
         }
     }
-}
```

Comment:
Looks like no changes to this file.

Reply:
Removed unnecessary namespace statements